fs-objects-multipart.go

```diff
@@ -100,7 +100,7 @@ func (fs fsObjects) CompleteMultipartUpload(bucket string, object string, upload
 	}
 
 	// Cleanup all the parts.
-	if err = cleanupUploadedParts(fs.storage, mpartMetaPrefix, bucket, object, uploadID); err != nil {
+	if err = cleanupUploadedParts(fs.storage, mpartMetaPrefix, bucket, object, uploadID, false); err != nil {
```

Comment:
it will be easier to read if you create a recursive := false and pass that variable.